client/lbclient/boot/replication.js

```diff
@@ -23,17 +23,33 @@ module.exports = function(client) {
   };
 
   // setup model replication
-  function sync(cb) {
-    if (client.network.isConnected) {
-      RemoteTodo.replicate(LocalTodo, function() {
-        LocalTodo.replicate(RemoteTodo, cb);
+  var since = {push: -1, pull: -1};
+  function sync() {
+    LocalTodo.replicate(
+      RemoteTodo,
+      since.push,
+      function pushed(err, conflicts, cps) {
+        console.log('LR', arguments);
```

Comment:
I think the problem is caused by the fact that your RemoteTodo (=base Todo ATM) has change tracking enabled, thus it's trying to rectify changes via REST API. That's why you need to create a new subclass RemoteTodo which will not track changes but still provide the replication API. See the docs for more details.

> For each replicated model, create two new client-only subclasses:
 - A local model that will use local storage to persist the changes offline
 - A remote model that will be connected to the server and used as a target for replication. This model will have change tracking disabled (because the server is already handling it) and enable only the replication REST API.